doc/decisions/STEPS.md

```diff
@@ -0,0 +1,86 @@
+# Steps
+
+This document describes all steps a decision can run through.
+
+```mermaid
+flowchart LR
+    s((Start)) --> Drafts --> In_Discussion --> In_Progress --> Decided --> Partially_Implemented --> Implemented
+
+    %% Shortcuts:
+    s --> Decided
+    Decided --> Implemented
+
+    %% Dead ends:
+    s --x Rejected
+    s --o Delayed
+    Drafts --x Rejected
+    Drafts --o Delayed
+    In_Discussion --x Rejected
+    In_Discussion --o Delayed
+    In_Progress --x Rejected
+    In_Progress --o Delayed
+```
+
+## Drafts
```

Comment:
```suggestion
The first PR for a decision must **always** create the decision in the "Draft" state at first.
If during the reviews of this PR it becomes clear that the decision is further along, can be moved to "In Discussion" or "Decided" before the merge.

## Drafts
```

IMO this is a better process, because it makes clear how the first PR should be created. Otherwise the decision author needs to guess what the correct initial state of the decision is.